### DISCUSSION TITLE
Question: max_epochs cannot be used.

### DISCUSSION THREAD

**Discussion by VivizSun** [open-edge-platform/anomalib#2479] [Q&A] [answered]
Thank you for your contributions.

When I train custom data with the training parameter max_epochs=20, it stops after only one epoch (as shown in the image below).
What settings are missing?

![螢幕擷取畫面 2024-12-23 165601](https://github.com/user-attachments/assets/37dfbde7-45ad-41f0-a7b2-e01ff6ed56cb)

**Comment by samet-akcay** [comment, marked as answer]
@VivizSun, you don't need to set `max_epochs` to train Patchcore. It only needs 1 epoch to extract the features from the data to form the memory bank. That's why it is set to 1, and cannot be changed.
